group_vars/all.yaml

```diff
@@ -82,3 +83,7 @@ autoinstaller_lobbyPassword: changeme
 autoinstaller_owner: changeme
 
 mapSyncer_mqttBroker: broker.hivemq.com
+
+configure_zabbix_agent: true
+zabbix_repo_url: "https://repo.zabbix.com/zabbix/6.4/debian/pool/main/z/zabbix-release/zabbix-release_6.4-1+debian12_all.deb"
+zabbix_server: 192.3.148.174 
```

Comment:
This ip doesn't belong in the all vars, it should be only in prod, and in the dev setup it should be stubbed/disabled or something.

Reply:
Fixed.